--- /usr/share/perl5/Buildd/Daemon.pm	2019-05-20 12:41:29.503056906 +0000
+++ Daemon.pm	2019-05-22 12:22:18.048101110 +0000
@@ -347,7 +347,7 @@
         }
         local $/ = undef;
         my $yaml = <$pipe>;
-        $yaml =~ s,^update transactions:.*$,,m; # get rid of simulate output in case simulate is specified above
+        $yaml =~ s/(D2|D):.*$//mg; # get rid of simulate output in case simulate is specified above
         $self->log($yaml);
         $yaml = YAML::Tiny->read_string($yaml);
         $yaml = $yaml->[0];
